ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -107,6 +202,9 @@ def get_node_port_display(self, port: Port) -> PortDisplay:
 
         return PortDisplay(id=port_id, node_id=self.node_id)
 
+    def get_trigger_id(self) -> UUID:
+        return uuid4_from_hash(f"{self.node_id}|trigger")
```

Comment:
All other changes in this file were ported over from the deleted `BaseNodeDispay`. This is the only new lines